### ISSUE TITLE
Mqtt root topic manager 

### ISSUE THREAD

**Issue by Lolagatorade** [meshtastic/Meshtastic-Apple#854]
### OS

iOS, iPadOS, macOS

### Description

As a general UX improvement for users it would be nice and easier to select your favorite mqtt topics from general settings and then from the primary channel choose which root topic you would like to have a conversation with. 

As I change root topics quite frequently in a day it would be nice to have a list of favorites ready to go from the actual chat drop down and checkmark the one I want. 

As of now it’s not user friendly if you want to change root topics you have to hunt it down in general setting whereas there is a very nice button on primary channel that shows the current root topic but is not user intractable. 

![IMG_7550](https://github.com/user-attachments/assets/2a8343d9-6fca-4885-bbf8-cf991ddfb65f)

### Participation

- [ ] I am willing to submit a pull request for this issue.

### Additional comments

![IMG_7551](https://github.com/user-attachments/assets/99297bf0-c316-4b6c-8541-8361083b3a6f)
In conclusion The improvement is: Configure favorite root topics from settings. And from the primary have a drop down on the current non-intractable dropdown where its is possible to checkmark the desired root topic
